cmds/core/ps/ps.go

```diff
@@ -214,32 +230,9 @@ func ps(pT *ProcessTable, w io.Writer) error {
 	return nil
 }
 
-func usage() {
-	log.Printf("Uage: ps [flags] [aux]")
-	flag.Usage()
-	os.Exit(1)
-}
-
 func main() {
 	flag.Parse()
-	// The original ps was designed before many flag conventions existed.
-	// It had switchwes not needing a -. Try to emulate that.
-	// It's pretty awful, however :-)
-	for _, a := range flag.Args() {
-		switch a {
-		case "aux":
-			flags.all, flags.aux = true, true
-		default:
-			usage()
-		}
-	}
-	pT := NewProcessTable()
-	if err := pT.LoadTable(); err != nil {
-		log.Fatal(err)
-	}
-
-	err := ps(pT, os.Stderr)
-	if err != nil {
+	if err := ps(flag.Args(), os.Stderr); err != nil {
```

Comment:
if you go with my suggestion above, this could be
ps(os.Sdterr, flag.Args()...)
I'm surprised that it writes to  Stderr? Should be Stdout?
This explains why 
ps | grep 
fails ... I wonder when that happened? A genuine bug, found in code review, who knew?